src/pal/src/exception/seh-unwind.cpp

```diff
@@ -85,6 +85,33 @@ static void WinContextToUnwindContext(CONTEXT *winContext, unw_context_t *unwCon
 #undef ASSIGN_REG
 }
 #else
+static void UpdateUnwindContextWithWinContext(CONTEXT *winContext, unw_context_t *unwContext)
+{
+#if defined(_ARM_)    
+    // Assuming that unw_set_reg() on cursor will point the cursor to the
+    // supposed stack frame is dangerous for libunwind-arm in Linux.
+    // It is because libunwind's unw_cursor_t has other data structure
+    // initialized by unw_init_local(), which are not updated by
+    // unw_set_reg().
+    unwContext->regs[0] = 0;
+    unwContext->regs[1] = 0;
+    unwContext->regs[2] = 0;
+    unwContext->regs[3] = 0;
+    unwContext->regs[4] = winContext->R4;
+    unwContext->regs[5] = winContext->R5;
+    unwContext->regs[6] = winContext->R6;
+    unwContext->regs[7] = winContext->R7;
+    unwContext->regs[8] = winContext->R8;
+    unwContext->regs[9] = winContext->R9;
+    unwContext->regs[10] = winContext->R10;
+    unwContext->regs[11] = winContext->R11;
+    unwContext->regs[12] = 0;
+    unwContext->regs[13] = winContext->Sp;
+    unwContext->regs[14] = winContext->Lr;
+    unwContext->regs[15] = winContext->Pc;
+#endif    
+} 
```

Comment:
``WinContextToUnwindCursor`` takes ``unw_cursor_t`` rather than ``unw_context_t``. We need to update the existing ``unw_context_t``. I think that it would be better to have another function responsible to update ``unw_context_t`` (instead of taking ``unw_context_t`` as an extra argument of ``WinContextToUnwindCursor``)